### PR TITLE
Code of Conduct update

### PR DESCRIPTION
**Pull request by ha0ye**
resolves uf-carpentry/coordination#100

### CODE REVIEW

**Comment by ha0ye** [comment]
Deploy preview for *uf-carpentries* ready!

Built with commit de7f99093e1a56dcc3527775697d630530d0bfa4

https://deploy-preview-45--uf-carpentries.netlify.com

File: _pages/code-of-conduct.md [hunk 1]
```diff
@@ -5,6 +5,86 @@ title: Code of Conduct
...
+
+## Enforcement
+
+Community members and workshop participants who are asked to stop any harassing or discriminatory behavior are expected to comply immediately. This applies to all of our workshops and events, as well as interactions that take place online, across e-mail, Etherpad, social media or in any other medium.
+
+If someone engages in behavior that violates this code of conduct, they may be warned, asked to leave the event or premises, or reported to other organizations for further investigation and sanctions (including, but not limited to The Carpentries Code of Conduct Committee, the University of Florida, or law enforcement). 
+
+If you are being harassed, notice that someone else is being harassed, or have any concerns, please let us know through one or more of these reporting options:
+
+**(Do note that some of contacts may be UF employees who are mandatory reporters. If you have concerns about privacy or escalation of certain incidents, you may wish to use the anonymous form or check with the instructor or organizer of the event before reporting.)**
```
Comment (gaurav) [inline _pages/code-of-conduct.md]: I'd love to link "mandatory reporters" to a link to UF's mandatory reporting policy, but I couldn't find it. The closest I got is our [sexual harassment policy](https://hr.ufl.edu/forms-policies/policies-managers/sexual-harassment/), which says: "Any employee, student, or visitor who has knowledge of an occurrence of sexual misconduct or questions regarding the policy is strongly encouraged to promptly reach out to the following party [...] Failure to comply with this policy could result in disciplinary action, up to and including disaffiliation or termination as deemed appropriate by the adjudicating body."

File: _pages/code-of-conduct.md [hunk 2]
```diff
@@ -5,6 +5,86 @@ title: Code of Conduct
...
+:x: do not insult, harass, or threaten violence
+
+**Have a concern?** Let one of the event organizers know, or use our [anonymous reporting form](***)
+
+Read our full Code of Conduct at: https://www.uf-carpentries.org/code-of-conduct/
+
+## License & Acknowledgments
+This document is distributed under a [Creative Commons Attribution-ShareAlike license](https://creativecommons.org/licenses/by-sa/3.0/).
+
+This document is adapted from multiple sources, including the [Carpentries Code of Conduct](https://docs.carpentries.org/topic_folders/policies/code-of-conduct.html), the [Davis R-Users Group Code of Conduct](https://d-rug.github.io/code-of-conduct.html), the [Mozilla Community Participation Guidelines](https://www.mozilla.org/en-US/about/governance/policies/participation/) and the [Weecology Lab Code of Conduct](https://github.com/weecology/lab-wiki/wiki/WEecology:-Code-of-Conduct).
```
[the remark this answers is not in the document]
Comment (ha0ye) [reply _pages/code-of-conduct.md]: Yep, private repo.

File: _pages/code-of-conduct.md (hunk 1 above)
Comment (ha0ye) [reply _pages/code-of-conduct.md]: I wasn't able to dig out a description either, but it's definitely a component of the required training for, I'm guessing staff (incl. postdocs) and faculty.

**Pull request merged**